dmc2gym/__init__.py

```diff
@@ -11,14 +11,15 @@ def make(domain_name,
          width=84,
          camera_id=0,
          frame_skip=1,
+         max_episode_steps_before_skip=1000,
```

Comment:
I think this should be called the episode_length instead. At least that the meaning I put in it.

Reply:
Updated